### ISSUE TITLE
MICN x_dec 被处理了两次吗？

### ISSUE THREAD

**Issue by Yongke** [thuml/Time-Series-Library#31]
File /storage/thumlbased/TimeSeriesLibrary/models/MICN.py:161, in Model.forecast(self, x_enc, x_mark_enc, x_dec, x_mark_dec)
    159 zeros = torch.zeros([x_dec.shape[0], self.pred_len, x_dec.shape[2]], device=x_enc.device)
    160 seasonal_init_dec = torch.cat([seasonal_init_enc[:, -self.seq_len:, :], zeros], dim=1)
--> 161 dec_out = self.dec_embedding(seasonal_init_dec, x_mark_dec)

在exp_long_term_forecasting.py中看起来已经有类似的处理，重复了吗？

**Comment by ZDandsomSP** [comment]
Thank you for your reminder. We noticed this problem in the process of code collation. Because of the model design of MICN, this part of code is reserved by us. We will adjust the relevant code according to your suggestions in the next commit
